package.json

```diff
@@ -10,7 +10,8 @@
     "bulma": "^0.9.3",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-scripts": "^4.0.3"
+    "react-scripts": "^4.0.3",
+    "uuid": "^9.0.0"
```

Comment:
In the future - Do not add extra files to PR
To add the file to a commit use git add src/.